### ISSUE TITLE
[SF][A11y][1.5.0][Toolbar]Toolbar overflow content does not have correct announcement when opening

### ISSUE THREAD

**Issue by ksblklu** [SAP/ui5-webcomponents-react#3793]
**Describe the bug**
Toolbar overflow content does not have correct announcement when opening

**To Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents-react/?path=/docs/layouts-floorplans-toolbar--with-overflow-button
![image](https://user-images.githubusercontent.com/27538066/203274117-cbee094a-a503-4136-b794-513b490d539b.png)
2. Click on overflow button of  Toolbar
![image](https://user-images.githubusercontent.com/27538066/203274278-66737968-148e-4b39-b474-74a97e061024.png)

3. Inspect JAWS speech history
<img width="1501" alt="image" src="https://user-images.githubusercontent.com/27538066/203280146-6f44e31b-bc73-4e10-831d-572d71d27ab1.png">

4. See there are not annoucement indicate the overflow menu was opened.

**Expected behavior**
Please see [WCAG reference](https://w3c.github.io/aria-practices/examples/menu-button/menu-button-actions-active-descendant.html) below
<img width="1514" alt="image" src="https://user-images.githubusercontent.com/27538066/203280532-12bdc074-61a8-413f-95a7-6616737f1279.png">
<img width="1673" alt="image" src="https://user-images.githubusercontent.com/27538066/203280610-d018ac72-0a79-44ab-a761-365d8ec3b6b4.png">

Upon inspecting, I find the role="dialog" to be present for the expanded menu section which is incorrect as the expanded section is not a dialog but a menu as seen in the WCAG reference

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 1.9.0
`@ui5/webcomponents-react` version: 1.5.0
Operating System:Windows
Browser: any

**Comment by Lukas742** [comment]
Hi @ksblklu 

> Expected behavior
> Please see WCAG reference below
Next time please link the article, so we don't have to search the web first, to get more information.

Since the popover can also contain non-actionable items like text, `menu` would be the wrong announcement in my opinion. 
If you want to customize it, you would first need to open a feature request, which enables changing the role of the `Popover` component in the [ui5-webcomponents](https://github.com/SAP/ui5-webcomponents/issues/new/choose) repo, as the component currently doesn't support changing the role and then you could set if by leveraging the `overflowPopoverRef`.

As you mentioned the role="dialog" is present for the popover, so this is probably a bug with JAWS, as for NVDA and VoiceOver (MacOS) the announcement is made.

**Comment by ksblklu** [comment]
Hi @Lukas742 

> Next time please link the article, so we don't have to search the web first, to get more information.
> 
Thanks for pointing this out, I've updated the article link above, and raised a feature request to [ui5-webcomponents](https://github.com/SAP/ui5-webcomponents/issues/6090).

**Comment by ksblklu** [comment]
Hi @Lukas742 

I noticed that currently we cannot access the overflow `Popover` of `DynamicPageTitle` via `overflowPopoverRef` , can we make it possible to access overflow Popover via `actionsToolbarProps - overflowPopoverRef`?
<img width="1372" alt="image" src="https://user-images.githubusercontent.com/27538066/204221121-7d0eb732-2c50-40cb-9498-9c676e502323.png">

Thanks

**Comment by Lukas742** [comment]
Hi @ksblklu 

after this [PR](https://github.com/SAP/ui5-webcomponents-react/pull/3805) is merged and released, you will be able to access the overflow popover instance of the respective `Toolbar` via the `overflowPopoverRef` property.

Since the original question has been resolved, I'm going to close this issue now.
